content/100-getting-started/index.mdx

```diff
@@ -1,108 +1,163 @@
-  color="dark"
-  type="primary"
-  href="./getting-started/setup-prisma/start-from-scratch/relational-databases-typescript-postgresql"
->
-  New SQL project
-</ButtonLink>
-
-<ButtonLink
-  color="dark"
-  type="primary"
-  href="./getting-started/setup-prisma/start-from-scratch/mongodb-typescript-mongodb"
->
-  New MongoDB project
-</ButtonLink>
+[**<Bolt />Prisma Accelerate**](/accelerate/what-is-accelerate) is a global database cache with scalable connection pooling.
 
-If you're interested in learning how Prisma integrates with your favorite library or framework, such as Next.js, Express, Apollo GraphQL, NestJS, etc, you can check out our **ready-to-run examples**.
+[**<SignalStream />Prisma Pulse**](/pulse/what-is-pulse) allows you to build reactive, real-time applications in a type-safe manner.
 
-<ButtonLink
-  color="dark"
-  type="primary"
-  href="https://github.com/prisma/prisma-examples/"
->
-  Examples
-</ButtonLink>
+</BorderBox>
 
-### Get started with Prisma Accelerate
-
-Add **Accelerate** to your app to use connection pooling and global database caching.
-
-<ButtonLink color="dark" type="primary" href="/accelerate/getting-started">
-  Get started with Accelerate
-</ButtonLink>
-
-### Get started with Prisma Pulse
-
-Add **Pulse** to your app to subscribe to real-time updates from your database using Prisma Client.
-
-<ButtonLink color="dark" type="primary" href="/pulse/getting-started">
-  Get started with Pulse
-</ButtonLink>
-
-## Prisma with different tooling
-
-Prisma can be used with a wide range of tooling and frameworks, the following links outline how to get started with some of them:
-
-- [Prisma with Next.js](https://www.prisma.io/nextjs)
-- [Prisma with GraphQL](https://www.prisma.io/graphql)
-- [Prisma with Apollo](https://www.prisma.io/apollo)
-- [Prisma with Nestjs](https://www.prisma.io/nestjs)
-- [Prisma with Express](https://www.prisma.io/express)
-- [Prisma with hapi](https://www.prisma.io/hapi)
-
-<Admonition type="info">
-
-Join our thriving community on [Slack](https://slack.prisma.io/), [Discord](https://discord.gg/KQyTW2H5ca), and [GitHub](https://github.com/prisma/prisma) for help and ideas.
+</TopBlock>
 
-</Admonition>
+## <Database color="#5A67D8" width="21" height="24" /> Prisma ORM
+
+### Easily explore Prisma ORM with a SQLite database
+
+_These options don't require you to have your own database server running._
+
+<Grid>
+  <LinkCard
+    icon="fa-solid fa-rabbit-running"
+    title="Quickstart"
+    desc="Set up Prisma ORM from scratch with a SQLite database in 5 minutes."
+    link="/docs/getting-started/quickstart"
+  />
+  <LinkCard
+    icon="fa-brands fa-github"
+    title="Examples"
+    desc="Explore our ready-to-run examples with your favorite frameworks and libraries."
+    link="https://github.com/prisma/prisma-examples/"
+  />
+</Grid>
+
+### Choose an option to get started with your own database
+
+_Select one of these options if you want to connect Prisma ORM to your own database._
```

Comment:
Note for other reviewers: This section didn't have the "Easily explore Prisma ORM with a SQLite database" heading and the italic sentences before. I added them because I felt like they're very helpful context to set expectations for each of the options that we offer here.

Reply:
Thanks for the note! Would simply "Explore with a simple SQLite database", or similar work also, to not repeat "Prisma ORM"?

Reply:
Updated to "Explore quickly with a SQLite database"